dbms/src/test/java/org/polypheny/db/misc/DataMigratorTest.java

```diff
@@ -157,25 +177,25 @@ public void alternativeCommandsTest() throws SQLException {
                             ImmutableList.of(
                                     new Object[]{ 1, 5, "foo" } ) );
 
-                    // Remove tvarchar column placement on initial store
+                    // Remove tvarchar column placement on initial storeId
```

Comment:
Looks like a find and replace error

Reply:
There are many more. I avoid creating a comment for every one.

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520476075